### DISCUSSION TITLE
Noob cluster example

### DISCUSSION THREAD

**Discussion by webfrank** [ergo-services/ergo#203] [Q&A]
Hi, I was wondering how to setup simple cluster with failover. I suppose to start say 3 nodes, one process will start an application with supervisor and some actors. Once everything is running the node crashes, does the cluster (supervisor) will take over to start the crashed actors on a different node? Is this feature available or is something should be implemented on top of the framework?

**Comment by halturin** [comment]
Hi Francesco,

There are a couple of options:
1. naive way - use Link/Monitor feature https://docs.ergo.services/basics/links-and-monitors
2. pro way - use Saturn https://docs.ergo.services/tools/saturn as a central registrar for all your nodes in the cluster. Saturn client https://docs.ergo.services/extra-library/registrars/saturn-slient allows you to be notified on a few cluster events (node join/left, app loaded/started/stopping/stopped, config update).

**Comment by halturin** [comment]
>does the cluster (supervisor) will take over to start the crashed actors on a different node?

current implementation of Supervisor doesn't support remote start, but you can handle such case by yourself using https://docs.ergo.services/networking/remote-spawn-process or https://docs.ergo.services/networking/remote-start-application
